docs/en/ufs/Ufs-Extension-API.md

```diff
@@ -45,12 +45,13 @@ to enable a storage system to serve as Alluxio's underlying storage.
 Step 1: Implement the interface `UnderFileSystem`
 
 The `UnderFileSystem` interface is defined in the module `org.alluxio:alluxio-core-common`. Choose
-to extend either `BaseUnderFileSystem` or `ObjectUnderFileSystem` to implement the `UnderFileSystem`
-interface. `ObjectUnderFileSystem` is suitable for connecting to object storage and abstracts away
+to extend either `ConsistentUnderFileSystem` or `ObjectUnderFileSystem` to implement the `UnderFileSystem`
+interface. `ConsistentUnderFileSystem` is used for storage like HDFS which does not have eventual consistency issue. 
```

Comment:
nit: make these 2 (consistent and object) bullets. 
which does not have eventual consistency issue -> which is not eventually consistent

Reply:
Done